CMakeLists.txt

```diff
@@ -490,6 +506,13 @@ endif()
 # Set CMAKE_INSTALL_* if not defined
 include(GNUInstallDirs)
 
+    
+if(${build_as_dynamic})
+    set(targets aziotsharedutil aziotsharedutil_dll)
+else(${build_as_dynamic})
```

Comment:
Could you please replace with else() which means same thing and is more readable to non-cmake experts.

Reply:
While I agree it makes it more readable to non-cmake experts I would defend this type of usage because in the event of mismatched if, elseif, else constructs first it is easier to see them and secondly cmake is very good at pointing out exactly where your error has occurred. I would advocate for using this construction for all if, elseif, else constructs but, if you really want me to, I will take it out.

---
In reply to: [184149662](https://github.com/Azure/azure-c-shared-utility/pull/152#discussion_r184149662) [](ancestors = 184149662)